tutorial/source/intro_part_ii.ipynb

```diff
@@ -586,30 +426,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
-   "metadata": {},
-   "outputs": [
```

Comment:
Same here..the output is stripped.

Reply:
nope none of that is intentional... i'm just entirely confused about nbstripout, how/when to keep cell output, etc

Reply:
We can run `nbstripout --keep-output` to retain the output or `make scrub`. 

@jpchen - Do we need any metadata, like `collapsed` for rendering the tutorials or can we strip out the metadata field completely?

Reply:
@martinjankowiak - for the time being though, you can revert these hunks manually using your git tool of choice :) (like SourceTree)

Reply:
well is the previous commit ok? the only difference between the two commits is nbstripout

Reply:
no i hardcoded the outputs where need be. just make sure the intro tutorials have that as wel

Reply:
@martinjankowiak - that looks fine except that we have some `collapsed: True` in the metadata.

@jpchen - So would I be correct in assuming that we do not need any metadata, but we should retain the output?

Reply:
yes

Reply:
oh god i tried to revert to 'small text changes' and then do make scrub but it didn't come out right... : (

Reply:
@martinjankowiak - Did `make scrub` not work? If all your changes are in and we have cruft like `execution count` etc. metadata, I can strip it out.

Reply:
well i think i got rid of all the text edits i had made in the first commit...